keyboards/handwired/wulkan/rules.mk

```diff
@@ -0,0 +1,25 @@
+MCU = STM32F303
+
+# Build Options
+#   comment out to disable the options.
+#
+BACKLIGHT_ENABLE = no
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+LAYOUTS = ortho_4x12
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+NO_UART = yes
+NO_SUSPEND_POWER_DOWN = yes
+UNICODEMAP_ENABLE = yes
+
+DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
```

Comment:
This isn't actually needed, as it's set later on. 

https://github.com/qmk/qmk_firmware/blob/master/quantum/stm32/proton_c.mk#L44


```suggestion
```


However, if you want to leave it here, you may want to set the DFU suffix, as well.

Reply:
Tested removing this one and it worked like a charm so will remove it :)